benchmark/messages.cpp

```diff
@@ -26,7 +26,25 @@ Moq_EncodeGroupHeader(benchmark::State& state)
 }
 
 static void
-Moq_EncodeGroupObject1000Bytes(benchmark::State& state)
+Moq_EncodeGroupHeader(benchmark::State& state)
+{
+    auto hdr = quicr::messages::MoqStreamHeaderGroup{};
+    hdr.subscribe_id = 0x100;
+    hdr.track_alias = 0x100;
+    hdr.priority = 0xA;
+    hdr.group_id = 0x1;
+
+    for ([[maybe_unused]] const auto& _ : state) {
+        quicr::Serializer buffer;
```

Comment:
I'd still pull the buffer out this loop just avoid measuring it, and only measure the encode. No need for the DoNotOptimize and ClobberMemory lines.